### DISCUSSION TITLE
Sunbird RC Implementation for End User Certificate Validation

### DISCUSSION THREAD

**Discussion by AmoghShet** [Sunbird-RC/community#754] [Ideas & Enhancements]
Hello everyone; I'd like to implement Sunbird RC to deploy a system where an admin issues certificates to users with a specific key/ID. Users can then go to a website portal, enter this ID/key there, & have that specific certificate pull up in a printable HTML format. Those certificates have to of course be cryptographically signed to ensure authenticity. I have a very basic RC backend set up. As a person with very little code experience, how feasibly easy is this to implement via Sunbird RC, and what would be the steps one would have to take to do so?

**Comment by Sreejit-K** [comment]
Hey Amogh! 

You can follow the below-mentioned steps for your use case. 

Please refer to this doc and let me know if you have any queries.  
[https://docs.google.com/document/d/1kzAXPzYthHaRI0oJKr2QZfZBb3PwgDwObDtiqIAwlqA/edit?usp=sharing]

cheers!
